recipes/default.rb

```diff
@@ -80,13 +80,15 @@
 end
 
 # Ensure the Kafka broker log (kafka data) directories exist
-node["kafka"]["server.properties"]["log.dirs"].split(",").each do |log_dir|
-  directory log_dir do
-    action :create
-    owner node["kafka"]["user"]
-    group node["kafka"]["group"]
-    mode 00700
-    recursive true
+if node["kafka"]["server.properties"].has_key? 'log.dirs'
```

Comment:
Kafka now has 2 log directory properties (log.dirs and log.dir). If log.dirs is not set it uses log.dir which has a default of /tmp/kafka